stores/searchResultsStore.ts

```diff
@@ -88,15 +88,12 @@ async function queryProfessionals(searchSpecialty?: Specialty, searchLanguage?:
             } satisfies HealthcareProfessionalSearchFilters
         }
 
-        console.log('searching professionals')
         const response = await gqlClient.request<{ healthcareProfessionals: HealthcareProfessional[] }>(searchProfessionalsQuery, searchProfessionalsData)
-        console.log(`Fetched professionals: ${JSON.stringify(response)}`)
 
         const professionalsSearchResult = (response?.healthcareProfessionals ?? []) as HealthcareProfessional[]
         return professionalsSearchResult
     } catch (error) {
         console.log(`Error getting professionals: ${JSON.stringify(error)}`)
-        alert(`Error getting data! Please contact our support team by clicking the bottom right link on the page!`)
```

Comment:
While it doesn't have to be an alert, we should let the user know when there's an error so they don't keep waiting and they can message us if they want. (Many people do surprisingly)

A nice toast message would be better here, but that's out of scope for this....how about keep it and replace ina new ticket?